css-layout-classes/styles.css

```diff
@@ -0,0 +1,17 @@
+.container {
+  width: 1000px;
+  margin: auto;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-fourth {
+  max-width: 250px;
+}
+
```

Comment:
let's add the missing classes you defined in your html doc (for two-thirds and one-third).